src/deployment/deploy.py

```diff
@@ -251,6 +251,24 @@ def check_region(self) -> None:
     def create_password(self, object_id: UUID) -> Tuple[str, str]:
         return add_application_password(object_id, self.get_subscription_id())
 
+    def get_url(self) -> str:
+        if self.multi_tenant_domain:
+            return "https://%s/%s" % (
+                self.multi_tenant_domain,
+                self.application_name,
+            )
+        else:
+            return "https://%s.azurewebsites.net" % self.application_name
+
+    def get_identifier_uri(self) -> str:
+        if self.multi_tenant_domain:
+            return "api://%s/%s" % (
+                self.multi_tenant_domain,
+                self.application_name,
+            )
+        else:
+            return "api://%s.azurewebsites.net" % self.application_name
```

Comment:
Can we link to the docs that define this format?

Reply:
i have not found a doc for this particular format, we derived it from the default value proposed by azure when we create the registration

Reply:
renamed and added comment